### DISCUSSION TITLE
Where I should put the bindI18n in next-i18next.config?

### DISCUSSION THREAD

**Discussion by Emauss** [i18next/next-i18next#2276] [Q&A] [answered]
I want to use reloadResources according to:
https://github.com/i18next/i18next-http-backend/tree/master/example/next#alternative-usage-the-preferred-way

And comment says:
"if all pages use the reloadResources mechanism, the bindI18n option can also be defined in next-i18next.config.js"
I'm using reloadResources in all pages and i want to put bindI18n into next-i18next.config.js

![image](https://github.com/i18next/next-i18next/assets/61393058/f9731c6a-e50a-4e54-b563-2d3c4a506654)

But in my case there's a warning about invalid config options.

![image](https://github.com/i18next/next-i18next/assets/61393058/49a89010-b0bd-408c-8fcf-1e80450389c2)

I'm putting this in right way? 
btw fallbackLng also is wrong according to console. 
Maybe it occurs because Next version is wrong or sth? Any ideas?

deps versions:
```
"next-i18next": "^15.2.0"
"react-i18next": "^14.1.0"
"next": "13.5.6"
"react": "^18.2.0"
```

**Comment by adrai** [comment]
bindI18n is in the react options: https://github.com/i18next/i18next-http-backend/blob/master/example/next/next-i18next.config.js#L15

**Reply by Emauss** [reply]
thank you for reply.
Should I take notice on these warnings in console from i18next like above?
Because also with react option this warning occurs.

**Reply by adrai** [reply]
I don't get that warning... I suspect you are using those options in the next.config.js file, instead of the next-i18next.config.js file...
And you also only need the i18n options in the next.config.js file

**Reply by adrai** [reply]
If you don't know what I mean, please provide your 2 config files

**Reply by Emauss** [reply]
It's for sure next-i18next.config.js.
And on next.config.js i have i18n options.
All translations on page works, but only warnings occurs and i don't know why

**Reply by adrai** [reply]
Please provide a minimal reproducible example, so I can check.

**Reply by Emauss** [reply]
One question, to use these options we need i18next-http-backend? Because this link is from this docs.

**Reply by adrai** [reply]
Normally yes

**Reply by Emauss** [reply]
Ok i'm not using http-backend for i18next so it could be explanation why i have this warnings.

**Reply by adrai** [reply, marked as answer]
If you bind the loaded event you need some sort of i18next backend... else it does not make much sense

**Reply by Emauss** [reply]
I tried solve another issue within this bind cause thought this could help, after reading thread about it.
Ok so i can close this discuss, thank you so much for help.
